sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -242,6 +242,8 @@ private JdbcIO() {}
     @Nullable
     abstract DataSource getDataSource();
 
+    abstract Boolean getPoolingDataSource();
```

Comment:
Are you sure it's supported by AutoValue ? AFAIR, we have to use `get` naming.

Reply:
In KafkaIO we have `boolean isEOS()`, so I think yes, it's supported.

Reply:
https://github.com/google/auto/blob/master/value/userguide/howto.md#beans
`Note that, in keeping with the JavaBeans specification, the is- prefix is only allowed on boolean-returning methods. get- is allowed on any type of accessor.`
So, I think it should be fine when since it returns boolean (not sure about boxing, perhaps we need return just primitive for this)

Reply:
Yes, I might use `boolean`.